docs/Installation.md

```diff
@@ -1,114 +1,221 @@
-# Setting up the kubernetes cluster
-
-This is a Documentation of the installation steps to setup an ovc cluster, please check the documentation for the script at [Installer-script](Installer-script.md)
-
-## Prerequisites
-
-- A JumpScale 9 installation(Preferably use docker image `openvcloud/js9`, otherwise see [docs](https://github.com/Jumpscale/bash) for installation)
-- Three nodes for deploying the cluster
-- Additional nodes for cpu/storage as required. 
-- For each node the user needs to have credentials to establish a SSH connection
+# OpenvCloud Cluster Setup
+
+Setting up an OpenvCloud cluster is done in following steps:
+- [Meet the prerequisites](#prerequisites)
+- [Create the configuration file](#create-config)
+- [Validate the configuration file](#validate-config)
+- [Configure the switches](#configure-switches)
+- [Install operation system on the controller nodes](#controller-os)
+- [Create and start the JumpScale 9 Docker container](#js9-container)
+- [Setup the Kubernetes cluster and deploy the OpenvCloud system containers](#kubernetes-cluster)
+- [Access the management container](#management-container)
+- [Install the operating systems on the nodes](#os-nodes)
+- [Setup the storage nodes](#storage-nodes)
+- [Install JumpScale services on nodes](#jumpscale-nodes)
+- [Deploy virtual machine images](#deploy-images)
+
+
+<a id="prerequisites"></a>
+## Meet the prerequisites
+
+- 3 dedicated controller nodes
+- 10 dedicated CPU nodes
+- 4 dedicated storage nodes
```

Comment:
this of course is only 1 of the suipported sizes

Reply:
updated worthing